ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -86,7 +86,8 @@ function build_push_operator_images {
   DOCKERFILE="index.Dockerfile"
   DOCKERFILE_PATH_PATCH=(\{\"spec\":\{\"strategy\":\{\"dockerStrategy\":\{\"dockerfilePath\":\""${DOCKERFILE}"\"\}\}\}\})
 
-  opm index add --bundles "${BASE_BUNDLE}" --out-dockerfile "${DOCKERFILE}" --generate
+# todo: Improve include manila bundle workflow
+  opm index add --bundles "${BASE_BUNDLE}",quay.io/openstack-k8s-operators/manila-operator-bundle:latest --out-dockerfile "${DOCKERFILE}" --generate
```

Comment:
This code is called two times: 1)  building op being testes (eg. Manila op) and 2) building openstack operator. 
Do you want to add manila-bundle on both? 
I think that we should, as a next step, use a logic from openstack-operator. In order to do that, we need a patch there that allows us to run `opm index add` without building/pushing the image.

Reply:
@viroel is there a particular reason to not rely on what is done in the `openstack-operator`?  e.g. we can run the script [1] (which is already used by the github action), then tag and push the current build to `rdo`.
The alternative I guess is calling the `Makefile` targets provided by the openstack-operator and build everything from scratch (like we do locally):

```
git clone <openstack-operator>
pushd <openstack-operator-repo>
export TAG=<hash from the repo>
export USER=<the rdo data>
export IMG=quay.io/$USER/openstack-operator:<tag>
export BUNDLE_IMG=quay.io/$USER/openstack-operator-bundle:<tag>

make docker-build && make docker-push && make bundle && make bundle-build && make bundle-push && make catalog-build && make catalog-push
 ```

Reply:
> @viroel is there a particular reason to not rely on what is done in the `openstack-operator`? e.g. we can run the script [1] (which is already used by the github action), then tag and push the current build to `rdo`. The alternative I guess is calling the `Makefile` targets provided by the openstack-operator and build everything from scratch (like we do locally):
> 
> ```
> git clone <openstack-operator>
> pushd <openstack-operator-repo>
> export TAG=<hash from the repo>
> export USER=<the rdo data>
> export IMG=quay.io/$USER/openstack-operator:<tag>
> export BUNDLE_IMG=quay.io/$USER/openstack-operator-bundle:<tag>
> 
> make docker-build && make docker-push && make bundle && make bundle-build && make bundle-push && make catalog-build && make catalog-push
> ```

With this strategy, if you get PR#243 in, you automatically build the `-index` in the way specified in the openstack-operator (and you don't have to change things twice)

Reply:
@viroel yeah, if we build keystone index(for keystone-operator PR), the current patchset code would means we are adding manila bundle in keystone index too. 

@fmount prow containers from where we execute our jobs are rootless containers and because of that we are building the containers this way.